elasticsearch/src/main/java/org/hibernate/search/elasticsearch/analyzer/impl/ScopedElasticsearchAnalyzerReference.java

```diff
@@ -63,6 +63,20 @@ public CopyBuilder startCopy() {
 		return new Builder( globalAnalyzerReference, scopedAnalyzerReferences );
 	}
 
+	@Override
+	public String toString() {
```

Comment:
This is a good idea, but don't you think we should also add the names of the targeted fields in `RemoteMatchQuery#toString()`? From what I see [here](https://forum.hibernate.org/viewtopic.php?f=9&t=1045140) the field name isn't printed out, and it seems like an even greater problem to me... We should probably have a look at `org.apache.lucene.search.TermQuery#toString(String)` and use the same principles in `RemoteMatchQuery#toString()`.

Reply:
Mmmh, yes, somehow missed it. Will fix too.